test/integration_test.js

```diff
@@ -0,0 +1,95 @@
+};
+let clientService;
+
+describe('Integration test on ClientService', async () => {
+  const mockedFunctionId = 'TestFunction';
+  const mockedContractId = `StateUpdater${Date.now()}`;
+  const mockedContractName = 'com.org1.contract.StateUpdater';
+  const mockedFunctionName = 'com.org1.function.TestFunction';
+  const mockedAssetId = 'mockedAssetId';
+  const mockedState = 1;
+  const mockedContractArgument = {
+    asset_id: mockedAssetId,
+    state: mockedState,
+  };
+  const mockedFunctionArgument = {
+    asset_id: mockedAssetId, // passed via contractArgument
+    state: mockedState,
+    _function_: mockedFunctionId,
+  };
+  const property = {
+    asset_id: mockedAssetId,
+    state: mockedState,
+  };
+  const mockedByteContract = fs.readFileSync(
+      __dirname + '/StateUpdater.class');
+  const mockedByteFunction = fs.readFileSync(
+      __dirname + '/TestFunction.class');
+  clientService = new ClientService(properties);
+  describe('registerCertificate', () => {
+    it('should works as expected', async () => {
+      const response = await clientService.registerCertificate();
+      assert.equal(response.getStatus(), 200);
+    });
+  });
+  describe('registerFunction', () => {
+    it('should works as expected', async () => {
+      const response = await clientService.registerFunction(mockedFunctionId,
+          mockedFunctionName,
+          mockedByteFunction);
+      assert.equal(response.getStatus(), 200);
+    });
+  });
+  describe('registerContract', () => {
+    it('should works as expected', async () => {
+      const response = await clientService.registerContract(mockedContractId,
+          mockedContractName,
+          mockedByteContract, property);
+      assert.equal(response.getStatus(), 200);
+    });
+  });
+  describe('listContracts', () => {
+    it('should works as expected', async () => {
+      const response = await clientService.listContracts(mockedContractId);
+      assert.equal(response.getStatus(), 200);
+    });
+  });
+  describe('executeContract', () => {
+    it('should works as expected when execute a registered Contract',
+        async () => {
+          const response = await clientService.executeContract(mockedContractId,
+              mockedContractArgument, property);
```

Comment:
If I am not wrong, the third parameter is not a ``property`` but the ``functionArgument``. See https://github.com/scalar-labs/scalardl-javascript-sdk-base/blob/063d4ec4fafca674b813b3176aa1ef03d8fdfa4c/index.js#L249